file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,592 @@
+INTERFACE zif_aff_cfdf_v1
+  PUBLIC.
+
+  TYPES: ty_field_name              TYPE c LENGTH 30.
+  TYPES: ty_component_identifier    TYPE c LENGTH 5.
+  TYPES: ty_busienss_context        TYPE c LENGTH 30.
+  TYPES: ty_cds_view_name           TYPE c LENGTH 30.
+  TYPES: ty_badi_implemenation_name TYPE c LENGTH 30.
```

Comment:
Just a heads up: For these you might want to use type `zif_aff_types_v1=>ty_object_name_30`

Reply:
`ty_cds_view_name` and `ty_badi_implemenation_name` have been changed, as they refer to real objects in the system. `ty_busienss_context` is however not a real repository object, but an internal term...